build_ngtf.py

```diff
@@ -145,9 +152,48 @@ def main():
         action="store",
         default='')
 
+    parser.add_argument(
+        '--build_base',
+        help="Builds a base ngtf image in docker\n",
+        action="store_true")
+
+    parser.add_argument(
+        '--start_container',
+        help="Starts the docker container\n",
+        action="store_true")
+
+    parser.add_argument(
+        '--stop_container',
+        help="Stops the docker container\n",
+        action="store_true")
+
+    parser.add_argument(
+        '--run_in_docker',
+        help="Runs build_ngtf.py in a Dockerfile\n",
+        action="store_true")
```

Comment:
would it be possible and more user friendly to use sub_parser/grouping to group the docker commands together?

Reply:
@avijit-nervana : should we add a buildkite CI pipeline to test the docker build at some point?